pom.xml

```diff
@@ -1,213 +1,218 @@
-        </repository>
-        <snapshotRepository>
-            <id>openmrs-repo-snapshots</id>
-            <name>OpenMRS Snapshots Repo</name>
-            <url>https://mavenrepo.openmrs.org/nexus/content/repositories/snapshots</url>
-        </snapshotRepository>
-    </distributionManagement>
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>org.openmrs.maven.plugins</groupId>
+	<artifactId>openmrs-packager-maven-plugin-parent</artifactId>
+	<version>1.8.0-SNAPSHOT</version>
+	<packaging>pom</packaging>
+
+	<name>OpenMRS Packager Maven Plugin Parent Project</name>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+	</properties>
+
+	<scm>
+		<connection>scm:git:git@github.com:openmrs/openmrs-contrib-packager-maven-plugin.git</connection>
+		<developerConnection>scm:git:git@github.com:openmrs/openmrs-contrib-packager-maven-plugin.git</developerConnection>
+		<url>scm:git:git@github.com:openmrs/openmrs-contrib-packager-maven-plugin.git</url>
+		<tag>HEAD</tag>
+	</scm>
+
+	<modules>
+		<module>maven-plugin</module>
+		<module>integration-tests</module>
+	</modules>
+
+	<profiles>
+		<profile>
+			<id>integration-tests</id>
+			<modules>
+				<module>integration-tests</module>
+			</modules>
+		</profile>
+	</profiles>
+
+	<!-- For plugin development -->
+	<dependencies>
+		<dependency>
+			<groupId>org.apache.maven</groupId>
+			<artifactId>maven-plugin-api</artifactId>
+			<version>3.2.1</version>
+			<exclusions>
+				<exclusion>
+					<groupId>com.google.guava</groupId>
+					<artifactId>guava</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+		<dependency>
+			<groupId>com.google.guava</groupId>
+			<artifactId>guava</artifactId>
+			<type>jar</type>
+			<version>31.0.1-jre</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.maven.plugin-tools</groupId>
+			<artifactId>maven-plugin-annotations</artifactId>
+			<version>3.2</version>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.codehaus.plexus</groupId>
+			<artifactId>plexus-utils</artifactId>
+			<version>3.0.8</version>
+		</dependency>
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>4.13.1</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.powermock</groupId>
+			<artifactId>powermock-module-junit4</artifactId>
+			<version>2.0.9</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.powermock</groupId>
+			<artifactId>powermock-api-mockito2</artifactId>
+			<version>2.0.9</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.maven.shared</groupId>
+			<artifactId>maven-verifier</artifactId>
+			<version>1.6</version>
+		</dependency>
+		<dependency>
+			<groupId>com.vdurmont</groupId>
+			<artifactId>semver4j</artifactId>
+			<version>3.1.0</version>
+		</dependency>
```

Comment:
Could we use [the maintained version of this library](https://central.sonatype.com/artifact/org.semver4j/semver4j)?

Reply:
Done! Validated the change (to use newer version) by executing test class.
(mvn test -Dtest=ValidateContentPackageMojo -DfailIfNoTests=false)